app/tests/spec/views/marketing_snippet.js

```diff
@@ -35,13 +37,28 @@ define(function (require, exports, module) {
     });
 
     describe('render', function () {
-      it('normally shows sign up marketing material to desktop sync users', function () {
+      it('shows marketing material users signing up to Sync on desktop', function () {
         windowMock.navigator.userAgent = 'Mozilla/5.0 (Windows NT x.y; rv:31.0) Gecko/20100101 Firefox/31.0';
 
         createView({
           language: 'en',
-          service: 'sync',
-          type: 'sign_up'
+          service: Constants.SYNC_SERVICE,
+          type: VerificationReasons.SIGN_UP
+        });
+
+        return view.render()
+          .then(function () {
+            assert.equal(view.$('.marketing.default').length, 1);
+          });
+      });
+
+      it('shows marketing material users signing in to Sync on desktop', function () {
```

Comment:
nit: Not really in scope of this PR, but didn't see a test for non english sync users. Should they see marketing?